ts/packages/ts/src/commands/commands.test.ts

```diff
@@ -222,4 +224,172 @@ describe(`commands`, () => {
+          configs: {
+            KEY0: 'KEY0',
+            KEY1: 'KEY1',
+          },
+        }).run();
+        const exportedConfigs = await new ExportCommand({ pipe: evalResult }).run();
+        expect(exportedConfigs).toStrictEqual({ KEY0: 'KEY0', KEY1: 'KEY1' });
+      });
+    });
+    test('Export with keys mutation callback', async () => {
+      const evalResult = await new EvalCommand({
+        store: store1,
+        set: set1,
+        schema: new ConfigSchema('mutate', {
+          KEY0: {
+            type: 'String',
+          },
+          KEY1: {
+            type: 'String',
+          },
+        }),
+        configs: {
+          KEY0: 'KEY0',
+          KEY1: 'KEY1',
+        },
+      }).run();
+      const exportedConfigs = await new ExportCommand({ pipe: evalResult, keys: (key) => `MY_${key}` }).run();
+      expect(exportedConfigs).toStrictEqual({ MY_KEY0: 'KEY0', MY_KEY1: 'KEY1' });
+    });
+    test('Export with bad keys mutation callback - returns non-string', async () => {
+      const evalResult = await new EvalCommand({
+        store: store1,
+        set: set1,
+        schema: new ConfigSchema('mutate', {
+          KEY0: {
+            type: 'String',
+          },
+          KEY1: {
+            type: 'String',
+          },
+        }),
+        configs: {
+          KEY0: 'KEY0',
+          KEY1: 'KEY1',
+        },
+      }).run();
+      await expect(
+        new ExportCommand({
+          pipe: evalResult,
+          // @ts-expect-error - should throw ConfigError
+          keys: (key) => ({ key }),
+        }).run(),
+      ).rejects.toBeInstanceOf(ConfigError);
+    });
+    test('Export with bad keys mutation callback - returns number', async () => {
+      const evalResult = await new EvalCommand({
+        store: store1,
+        set: set1,
+        schema: new ConfigSchema('mutate', {
+          KEY0: {
+            type: 'String',
+          },
+          KEY1: {
+            type: 'String',
+          },
+        }),
+        configs: {
+          KEY0: 'KEY0',
+          KEY1: 'KEY1',
+        },
+      }).run();
+      await expect(
+        new ExportCommand({
+          pipe: evalResult,
+          // @ts-expect-error - should throw ConfigError
+          keys: (key) => 5,
+        }).run(),
+      ).rejects.toBeInstanceOf(ConfigError);
+    });
+    test('Export with bad keys mutation callback - returns empty string', async () => {
+      const evalResult = await new EvalCommand({
+        store: store1,
+        set: set1,
+        schema: new ConfigSchema('mutate', {
+          KEY0: {
+            type: 'String',
+          },
+          KEY1: {
+            type: 'String',
+          },
+        }),
+        configs: {
+          KEY0: 'KEY0',
+          KEY1: 'KEY1',
+        },
+      }).run();
+      await expect(
+        new ExportCommand({
+          pipe: evalResult,
+          keys: (key) => '',
+        }).run(),
+      ).rejects.toBeInstanceOf(ConfigError);
```

Comment:
not sure is it returning an empty string or throwing an error?
`''` is not in the NAME `RESERVED_NAMES` list